### DISCUSSION TITLE
How to set a BackdropFilter for when the SpeedDial is active?

### DISCUSSION THREAD

**Discussion by banderberg** [darioielardi/flutter_speed_dial#239] [Ideas]
It seems that all I can do is set activeBackgroundColor but what I would really like to do is make the background covered by a BackdropFilter so I can use a blur effect.  Can this be done without forking the repo and modifying?

**Comment by luca-colazzo** [comment]
It can't be done now, but if you want to make it possible, any PR is welcome
